source/api/extension/text-granularity/index.md

```diff
@@ -0,0 +1,158 @@
+
+### 1.2 Motivating Use Cases
+
+A number of common workflows can result in annotation sets with discrete levels of text granularity:
+
+- OCR text is frequently available at a range of granularities, from page to character.
+- Manual transcription user interfaces may constrain user input to the level of the line,
+block, or page.
+- Transcriptions may be produced without coordinate data and thus have a very coarse level of granularity, such as page- or block-level.
+
+Identification of the level of text granularity in published annotations can facilitate the reuse of their textual content and target regions in other applications. Search is a primary use case, with the word-level annotations allowing for the accurate highlighting of search result terms in the user interface. Software designed for user interaction and input, such as an application designed to allow users to correct OCR text, might suppress finer-grained levels and make use of paragraph- or block-level text to simplify editing.
+
+## 2. Text Granularity Levels and the `textGranularity` Property
+
+The `textGranularity` property identifies the Text Granularity Level of a resource. The value _MUST_ be a single string. This extension defines the Text Granularity Levels found in the table below. The string _SHOULD_ be one of those defined in the table below or in the extensions registry.
+
+| Text Granularity Level       |  Description   |
+|------------------------------|-----------------
+| `page`                       |  A page in a paginated document |
+| `block`                      |  An arbitrary region of text    |
+| `paragraph`                  |  A paragraph                    |
+| `line`                       |  A topographic line             |
+| `word`                       |  A single word                  |
+| `glyph`                      |  A single glyph or symbol       |
+{: .api-table #table-granularity-level-dfn}
+
+```json-doc
+{ "textGranularity": "line" }
+```
+
+## 3. Use of the `textGranularity` Property with Annotations
+
+An Annotation _MAY_ have the `textGranularity` property. An Annotation that has the property _SHOULD_ target an IIIF Presentation API Canvas or segment and the identified Text Granularity Level _SHOULD_ describe that of the textual content represented by the content resources painted on the Target.
+
+The Annotation Body’s textual content _SHOULD_ be equivalent to the textual content represented by the content resources painted on the Target. For example, the Body of the Annotation might be a [TextualBody](https://www.w3.org/TR/annotation-model/#embedded-textual-body) that contains the transcription of the Target, which is painted with the image of a page of a medieval manuscript.
```

Comment:
> ..._SHOULD_ be equivalent...

What about _logically_ equivalent? Or some other way of making it clearer that, e.g., the text and the image should represent the same resource. I'm not sure that the "For example..." really captures it.

Reply:
There are a few inline links in the doc that should go in `links.md`

Reply:
Regarding "equivalent", I've created an issue since this is more than just a gloss and should probably be reviewed by the TRC.  https://github.com/IIIF/api/issues/1889   I'd suggest we keep the current text, since the document isn't semantically versioned, and adjust once we have agreed on a suggested change.

I think the links are working, some break because they reference the community part of the site and so aren't present in the branch preview.